### PR TITLE
docs(examples): fix in webpack1 example

### PR DESCRIPTION
**Pull request by layershifter**

### CODE REVIEW

File: examples/webpack1/.babelrc [hunk 1]
```diff
@@ -5,7 +5,6 @@
     "stage-1"
   ],
   "plugins": [
-    "lodash",
-    ["lodash", { "id": "semantic-ui-react" }]
+    ["lodash", { "id": ["lodash", "semantic-ui-react"] }]
```
Comment (layershifter) [inline examples/webpack1/.babelrc:8]: We [can](https://github.com/lodash/babel-plugin-lodash#usage) run resolution updates in single run.

File: examples/webpack1/public/index.html [hunk 2]
```diff
@@ -8,6 +8,6 @@
 </head>
 <body>
 <div id="root" style="height: 100%"></div>
-<script src="./dist/bundle.js"></script>
+<script src="./dist/app.js"></script>
```
Comment (layershifter) [inline examples/webpack1/public/index.html:11]: Fixed filename

File: examples/webpack1/src/main.js [hunk 3]
```diff
@@ -17,4 +17,4 @@ const App = () => (
   </Container>
 )
 
-render(App, MOUNT_NODE)
+render(<App />, MOUNT_NODE)
```
Comment (layershifter) [inline examples/webpack1/src/main.js:20]: Ugly mistake :sob:

**Comment by levithomason** [comment]
Whoops, I should have caught these issues in review!

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.68.1`.

**Pull request merged**